src/unity/Assets/Crest/Scripts/LodData/LodDataFlow.cs

```diff
@@ -1,38 +1,82 @@
 ﻿// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
 
 using UnityEngine;
+using UnityEngine.Rendering;
 
 namespace Crest
 {
     /// <summary>
     /// A persistent flow simulation that moves around with a displacement LOD. The input is fully combined water surface shape.
     /// </summary>
-    public class LodDataFlow : LodDataPersistent
+    public class LodDataFlow : LodData
```

Comment:
ahh you moved it to loddata! that seems like the right way to go for now, good stuff